### PR TITLE
Attempt to get docgen output file(s) pushed back

### PR DESCRIPTION
**Pull request by gpdf**

### CODE REVIEW

File: .github/workflows/docgen_from_Jira.yaml [hunk 1]
```diff
@@ -39,7 +39,8 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
-          git commit -m "Update from JIRA." DMTR-301.tex DMTR-301.appendix.tex 
+          git commit -m "Update from JIRA." DMTR-301.tex DMTR-301.appendix.tex
```
Comment (jonathansick) [inline .github/workflows/docgen_from_Jira.yaml]:
```suggestion
          git commit -m "Update from JIRA" DMTR-301.tex DMTR-301.appendix.tex
```

File: .github/workflows/docgen_from_Jira.yaml [hunk 2]
```diff
@@ -39,7 +39,8 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
```
Comment (jonathansick) [inline .github/workflows/docgen_from_Jira.yaml]:
```suggestion
          git config user.email $GITHUB_ACTOR@users.noreply.github.com
```

File: .github/workflows/docgen_from_Jira.yaml [hunk 3]
```diff
@@ -39,7 +39,8 @@ jobs:
         run: |
           git config user.name github-actions
```
Comment (jonathansick) [inline .github/workflows/docgen_from_Jira.yaml]:
```suggestion
          git config user.name $GITHUB_ACTOR
```

**Review by jonathansick — APPROVED** [review]
Looks reasonable. I'm interested to see if there are any interactions with running this workflow and branch protections/required reviews if running from main or a tag. I do recommend updating the git user settings to something that uses a more appropriate email address and that identifies the individual that ran the workflow.

File: .github/workflows/docgen_from_Jira.yaml [hunk 4]
```diff
@@ -37,9 +37,9 @@ jobs:
  
       - name: Create commit
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-          git commit -m "Update from JIRA." DMTR-301.tex DMTR-301.appendix.tex
+          git config user.name $GITHUB_ACTOR
```
Comment (womullan) [inline .github/workflows/docgen_from_Jira.yaml:40]: @jonathansick so this should go in templates as well ?

**Review by womullan — APPROVED** [review]

File: .github/workflows/docgen_from_Jira.yaml (hunk 4 above)
Comment (jonathansick) [reply .github/workflows/docgen_from_Jira.yaml:40]: Yes, I think that'd be good.

**Comment by gpdf** [comment]
Closing this PR in favor of #6 .

**Pull request closed without merging**